src/sentry/integrations/jira/integration.py

```diff
@@ -32,3 +34,17 @@ class JiraIntegration(Integration):
 
     def get_pipeline_views(self):
         return []
+
+    def get_client(self, instance):
```

Comment:
yeah, i'd think we'd want it to represent the installed integration because otherwise, would we just put everything on the integration model? since at some point a lot of this might be abstracted into somethign like `IssueIntegration` or something, that doesn't seem like a viable option.